routing/pathfind.go

```diff
@@ -355,8 +355,8 @@ type graphParams struct {
 
 	// additionalEdges is an optional set of edges that should be
 	// considered during path finding, that is not already found in the
-	// channel graph.
-	additionalEdges map[route.Vertex][]*models.CachedEdgePolicy
+	// channel graph. Can be blinded edges too.
```

Comment:
nit: expand comment to note that these are either bolt 11 hints or blinded edges.